client/src/assets/blog-posts/index.js

```diff
@@ -1,5 +1,3 @@
 import { metadata as firstPostMetdata } from "./first-post/index.mdx";
 import { metadata as outreachyExperienceMetadata } from "./outreachy-experience/index.mdx"
-export default [
-    firstPostMetdata,outreachyExperienceMetadata
-]
+export default [firstPostMetdata,outreachyExperienceMetadata]
```

Comment:
change this format to this for readability and proper linting please.
```suggestion
export default [
  firstPostMetdata,
  outreachyExperienceMetadata
]
```